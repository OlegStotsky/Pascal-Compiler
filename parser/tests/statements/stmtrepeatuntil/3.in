var
	i : integer;
	a : integer;

begin
	a := 0;
	i := 0;
	repeat 
		a := a + i;
		i := i + 1;
	until i;
end;
