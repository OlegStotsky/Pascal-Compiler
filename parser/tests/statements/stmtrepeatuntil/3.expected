Error at line 11, column 3 : boolean expected but integer found
